README.md

```diff
@@ -47,8 +47,13 @@ Once you've picked a post-it, open the issue and read the details in full. At th
 
 * `pak::pak()` to make sure you've got the necessary packages.
 
-* `devtools::check()` to get baseline, hopefully all is well. If it isn't, 
-  seek help.
+* If you're working a documentation issue, run `devtools::document()` to
```

Comment:
```suggestion
* If you're working on a documentation issue, run `devtools::document()` to
```

Reply:
For the true novices, it's possible we should word this a bit more explicitly. Something about "regenerating the `.Rd` files maybe.